src/model/constants.rs

```diff
@@ -1,14 +1,21 @@
-
+// Model constants
 pub static MULTIPLIER: f64 = 45.0;
 pub static SIGMA: f64 = (25.0 / 5.0) * MULTIPLIER;
 pub static MU: f64 = 15.0 * MULTIPLIER;
 pub static TAU: f64 = SIGMA / 100.0;
 pub static BETA: f64 = SIGMA / 2.0;
 pub static KAPPA: f64 = 0.0001;
 pub static DECAY_DAYS: u64 = 115;
-pub static DECAY_MINIMUM: f64 = 825.0;
+pub static DECAY_MINIMUM: f64 = MULTIPLIER * 18.0;
 pub static DECAY_RATE: f64 = 0.06 * MULTIPLIER;
 
+// Initial rating constants
+pub static RATING_INTERCEPT: f64 = 45.0;
+pub static RATING_SLOPE: f64 = 3.2;
+pub static RATING_FLOOR: f64 = 5.0;
+pub static RATING_CIELING: f64 = 30.0;
```

Comment:
related question: how are different gamemodes currently being handled? these four constants will likely be different for taiko, catch, and mania (or eventually instead of hardcoding them we might want a function that calculates them), so it might be good to store them as an array of 4 numbers if you're passing in a player ID along with their gamemode

Reply:
In this app, I'm pretty sure nothing is handling different game modes rn. We do have access to the user-submitted tournament mode currently and can change things based on that. We can also have a function for these if you think that's best. Not sure what you mean by "passing in a player ID" because in this app, we have direct access to mu and sigma for each player. Maybe some pseudocode would help me picture what you're saying better.

Reply:
ah, I mean that each user may have different sets of rating histories for different gamemodes -- for example, I have some standard and some taiko tournaments in the database. So when calculating initial ratings, the standard one would use the formula we have here, but the taiko one would need to use a different RATING_INTERCEPT, so we might wanna store those constants here as well

Reply:
Ah okay. Sure, I can address that. What constants should be variable? Gonna mark them with a `STANDARD_` prefix to denote this.

Reply:
For now, probably just these four (rating_intercept, rating_slope, rating_floor, and rating_ceiling). It's possible we'll want some of the other constants to be different between modes as well (specifically default sigma), but I think we need a lot more data to tell.